frontend/build.yaml

```diff
@@ -15,11 +15,31 @@ targets:
           include:
             - "lib/build_config/build_config.yaml"
         enabled: True
-      df_protobuf:
-        generate_for:
-          include:
-            - "card.proto"
-        enabled: True
+      protoc_builder:
+        options:
+          # The version of the Protobuf compiler to use.
+          protobuf_version: "27.5"
+          # The version of the Dart protoc_plugin package to use.
```

Comment:
Should we really keep all these comments?

Reply:
I prefer to keep them, but removed them, since steffen also suggested removing them :D